### PR TITLE
Feature/get repos from a organization

### PR DESCRIPTION
**Pull request by kilwa0**
Hi,

Added the repository list from a organization into the data_source_github_organization.go, useful to automate changes into repositories organization

Regards

### CODE REVIEW

**Review by jcudit — COMMENTED** [review]
Added some suggestions around naming of the `repo` field to stay consistent with other resources in the project.  I'm having difficulties getting tests to pass locally but plan to revisit this by end of week.  Here is the error I am currently stuck on when running :

```sh
$ TF_LOG=DEBUG TF_ACC=1  go test -v   ./... -run ^TestAccGithubOrganizationDataSource
...
testing.go:654: Step 0 error: Check failed: Check 2/5 error: data.github_organization.test: Attribute 'name' expected to be set
```

File: github/data_source_github_organization.go [hunk 1]
```diff
@@ -32,6 +32,13 @@ func dataSourceGithubOrganization() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"repos": {
```
[the remark this answers is not in the document]
Comment (kilwa0) [reply github/data_source_github_organization.go:35]: Agree  about the changes of course :+1:

**Comment by kilwa0** [comment]
> Added some suggestions around naming of the `repo` field to stay consistent with other resources in the project. I'm having difficulties getting tests to pass locally but plan to revisit this by end of week. Here is the error I am currently stuck on when running :
> 
> ```shell
> $ TF_LOG=DEBUG TF_ACC=1  go test -v   ./... -run ^TestAccGithubOrganizationDataSource
> ...
> testing.go:654: Step 0 error: Check failed: Check 2/5 error: data.github_organization.test: Attribute 'name' expected to be set
> ```
Hi!

Thank you for your time and your patience

To pass the test you just need to name the organization you are using for testing purposes in the organization settings page or remove the test, I've just kept the old check there 

Regards

**Comment by kfcampbell** [comment]
I've spent a little bit of time investigating this and I can confirm @kilwa0's assertion. I was able to get the tests to pass after providing an "Organization display name" and "Description" in previously-blank fields of my testing organization. Here's a screenshot of my organization settings page now:

image

That being said, the solution to this error isn't very discoverable. Perhaps we could flesh out [this section of CONTRIBUTING.md](https://github.com/integrations/terraform-provider-github/blob/master/CONTRIBUTING.md#github-organization) in order to start a list of prerequisites for organization setup. Or maybe we could provide a script that automates a test organization setup (it could even use this project itself).

@jcudit I think this is something of a product decision...taking this makes our tests a bit more brittle, which we could potentially mitigate with additional documentation or automation. What do you think?

**Comment by jcudit** [comment]
Lets add this to our docs.  I'd ❤️  a script that automates a test organization setup one day, but feel we can get by with just updating CONTRIBUTING.md for now.  I'll follow up with that separate to this PR as tests are now passing locally for me thanks to everyone's guidance 🙇🏾 .

**Review by jcudit — APPROVED** [review]

**Pull request merged**